package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pooltogether/etherplex",
-  "version": "0.2.3",
+  "version": "0.3.0",
```

Comment:
This is a major change!  This merits a major version update according to semver rules.

Let's push it to 1.0.0!